tests/cases.py

```diff
@@ -2429,6 +2429,7 @@ def test_combination_label(self):
 
     def test_combination_id(self):
         """Test combination without labels."""
+        # FIXME this test is failing in tests/test_datasets/test_combination
```

Comment:
Tests in `tests/test_datasets/test_combination.py` are now failing

Reply:
Looks like the results were actually wrong for `CoreTriplesFactory` inputs so far 😮